src/style.css

```diff
@@ -1,3 +1,81 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
 body {
   margin: 0;
 }
+
+header {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  background-color: #fff;
+  padding-inline: 50px;
+  align-items: center;
+  position: relative;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  align-items: center;
+}
+
+.nav {
+  margin: 0;
+  padding: 0;
+}
+
+.nav__link {
+  display: flex;
+  color: #000;
+  text-decoration: none;
+  font-size: 12px;
+  line-height: 60px;
+  align-items: center;
+  position: relative;
+}
+
+.nav__link.is-active {
+  color: #00acdc;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link:hover {
+  text-decoration-color: #00acdc;
+  color: #00acdc;
+}
+
+img {
```

Comment:
It's not recommended to style HTML elements directly (except for `html` and `body`). Instead, you should add a class to the `img` element and style that class. (CSS checklist)